src/Microsoft.ML.Transforms/CategoricalCatalog.cs

```diff
@@ -22,19 +22,19 @@ public static class CategoricalCatalog
         /// <param name="outputColumn">The output column. If <c>null</c>, <paramref name="inputColumn"/> is used.</param>
         /// <param name="outputKind">The conversion mode.</param>
         /// <returns></returns>
-        public static CategoricalEstimator OneHotEncoding(this TransformsCatalog.CategoricalTransforms catalog,
```

Comment:
>OneHotEncoding [](start = 43, length = 14)

`OneHotEncoder` as per guidelines

Reply:
I'll do OneHotEncodingEstimator, since we are using 'ing' rather than 'or'

---
In reply to: [227051943](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227051943) [](ancestors = 227051943)

Reply:
What I meant is `public static OneHotEncodingEstimator OneHotEncoder(this TransformsCatalog.CategoricalTransforms catalog`

---
In reply to: [227158077](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227158077) [](ancestors = 227158077,227051943)